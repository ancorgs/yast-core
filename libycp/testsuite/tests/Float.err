Parsed:
----------------------------------------------------------------------
300.000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
300.000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.0900000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
300.000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.0256000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
3.00000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
2.56000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.00000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.00000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0.00000000000000
----------------------------------------------------------------------
