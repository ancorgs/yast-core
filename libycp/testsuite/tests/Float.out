(300.000000000000)
(300.000000000000)
(0.0900000000000000)
(300.000000000000)
(0.0256000000000000)
(3.00000000000000)
(2.56000000000000)
(0.00000000000000)
(0.00000000000000)
(0.00000000000000)
